src/main/java/seedu/bookmark/ui/HelpWindow.java

```diff
@@ -15,17 +18,66 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL = "https://ay2021s1-cs2103t-f13-2.github.io/tp/";
+    public static final String HEADER_MESSAGE = "Welcome to bookmark!\n"
+                                                + "Click on the Copy URL button for the link to our Website!\n";
+
+    public static final String ADD_INTRO = "You can now ADD a book! \n";
+    public static final String ADD_COMMAND = "add n/{Book Name} g/{Genre} t/{Tags}* t/{Tags}* "
```

Comment:
Perhaps using the format provided in the [User Guide](https://github.com/AY2021S1-CS2103T-F13-2/tp/blob/master/docs/UserGuide.md) e.g. `add n/BOOK_TITLE tp/TOTAL_PAGES b/PAGE_NUMBER` would be more consistent?